lib/ui/widgets/state_responsive.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
```

Comment:
This can be replaced with import 'package:flutter/widgets.dart';

Reply:
Nice catch